zeppelin-web/pom.xml

```diff
@@ -120,34 +152,77 @@
           </execution>
 
           <execution>
-            <id>yarn install</id>
-            <goals>
-              <goal>yarn</goal>
-            </goals>
-            <configuration>
-              <arguments>install --no-lockfile</arguments>
-            </configuration>
+              <id>yarn add bower</id>
+              <goals>
+                  <goal>yarn</goal>
+              </goals>
+              <configuration>
+                <!--ignore-scripts-->
+                  <arguments>add bower --no-lockfile --no-bin-links</arguments>
```

Comment:
@Leemoonsoo, @1ambda  Oh, maybe this was misunderstood. In this context I meant environment which already used UI contributors and their scripts were described in "scripts" in package.json. I Didn't want to broke their usual style of build and test. I have done this only for compatibility with their using of CLI.
@Leemoonsoo Yes duplication Is not good practice
We can get a rid of these "scripts" and run all operations via maven.
e.g. for run build we can run 
```
mvn compile -Dmaven.clean.skip=true
```
Or if we want to run tests:
```
mvn test -Dmaven.clean.skip=true
```
or if we want just create package:
```
mvn package -Dmaven.clean.skip=true -Dmaven.test.skip=true
```
Installation of yarn, bower etc. I can put in phase "post-clean". So use -Dmaven.clean.skip=true can save a lot of time for installation of nodejs, yarn and bower 
@1ambda @Leemoonsoo Is this variant suit for you?